nifi/1.10/0.1.0-1.5.0/install/index.md

```diff
@@ -0,0 +1,300 @@
+---
+post_title: Install and Customize
+menu_order: 20
+enterprise: 'no'
+---
+
+ DCOS Nifi is available in the Universe and can be installed by using either the web interface or the DC/OS CLI.
+
+The default DC/OS NiFi Service installation provides reasonable defaults for trying out the service, but that may not be sufficient for production use. You may require different configurations depending on the context of the deployment.
+
+
+## Configuration Best Practices for Production
```

Comment:
Create a separate section called `Deployment Best Practices` and move the production configuration details there. Also, please expand all prod. deployment best practice. 
For reference: https://docs.mesosphere.com/services/percona-mongo/0.2.0-3.4.13/deploy-best-practice/

Reply:
Deployment Best Practice section has been added as suggested.